.github/workflows/push-on-develop.yml

```
@@ -35,19 +44,71 @@
+          env:
+            SSH_PRIVATE_KEY: ${{ secrets.PREPRODUCTION_SERVER_SSH_PRIVATE_KEY }}
+            SSH_USER: ${{ secrets.PREPRODUCTION_SERVER_SSH_USER }}
+            SSH_SERVER_IP: ${{ secrets.PREPRODUCTION_SERVER_SSH_ADDRESS }}
+            UPDATE_SCRIPT_FULL_PATH: ${{ secrets.PREPRODUCTION_UPDATE_SCRIPT_FULL_PATH }}
+          run: |
+            eval $(ssh-agent -s)
+            echo "$SSH_PRIVATE_KEY" | ssh-add -
+
+            ssh -o StrictHostKeyChecking=no $SSH_USER@$SSH_SERVER_IP << EOF
+              $UPDATE_SCRIPT_FULL_PATH
+            EOF
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)